get_coverage.py

```
@@ -0,0 +1,54 @@
+
+    total_found = sum(f['found'] for f in files.values())
+    total_hit = sum(f['hit'] for f in files.values())
+
+    print(f"Total Lines: {total_found}")
+    print(f"Covered Lines: {total_hit}")
+    print(f"Uncovered Lines: {total_found - total_hit}")
+    if total_found > 0:
+        print(f"Coverage: {total_hit / total_found * 100:.2f}%")
+
+    file_list = []
+    for filepath, data in files.items():
```

Ruff (0.15.9)

[warning] Redefining argument with the local name `filepath`

(PLR1704)